kinetic/distribution.yaml

```diff
@@ -1795,6 +1795,16 @@ repositories:
       url: https://github.com/ros/diagnostics.git
       version: indigo-devel
     status: maintained
+  dialogflow_ros:
+    doc:
+      type: git
+      url: https://github.com/piraka9011/dialogflow_ros.git
+      version: kinetic-devel
```

Comment:
CI fails because there is no branch named `kinetic-devel` on that repository (same below)

Reply:
This has been fixed, can you try again?